pkg/vex/vex_test.go

```diff
@@ -266,3 +278,23 @@ func TestOpenCSAF(t *testing.T) {
 		require.Len(t, doc.Statements, tc.len)
 	}
 }
+
+func TestOpen(t *testing.T) {
```

Comment:
Should we have a test for the new, updated version of the spec? Including an instructive JSON example of the current spec would also be helpful for orienting to what's expected in the current spec, too

Reply:
Good call, I've added a document to the test fixtures and added the test case her and a new (simple while I write a conformance suite) to the `Parse()` function.

Reply:
Okay, great! I think that'll be really nice to have. Over time, it'd be nice to expand the test data to cover more of the spec, too.